std_execution.bs

```diff
@@ -2805,16 +2805,17 @@ namespace std::execution {
     concept <i>class-type</i> = <i>decays-to</i>&lt;T, T> && is_class_v&lt;T>;  // exposition only
 
   // [execution.queries], general queries
-  inline namespace <i>unspecified</i> {
+  namespace <i>unspecified</i> {
```

Comment:
It would be better to be clear about the fact that these types occupy their _own_ namespace, different from all the others.

```suggestion
  namespace <i>general-queries</i> { <i>// Exposition only</i>
```

Please make this change everywhere.